tests/utils/test_triangulation.py

```diff
@@ -0,0 +1,32 @@
+"""Unit tests for triangulation utils.
+
+Authors: Ayush Baid
+"""
+import unittest
+
+import numpy as np
+from gtsam import PinholeCameraCal3Bundler, Pose3, Rot3
+
+import gtsfm.utils.triangulation as triangulation_utils
+
+
+class TestTriangulationUtils(unittest.TestCase):
+    def test_calculate_triangulation_angle_in_degrees(self):
+        """Test the computation of triangulation angle using a simple example."""
+
+        camera_center_1 = np.array([0, 0, 0])
+        camera_center_2 = np.array([10, 0, 0])
+        point_3d = np.array([5, 0, 10])
+
+        expected = 53.1301024
```

Comment:
53 degrees is a bit confusing, could we do a simpler one we can do in our head, like 45 deg?